src/content/docs/apis/nerdgraph/examples/multitenancy-accounts-nerdgraph.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: "NerdGraph tutorial: manage multi-tenant organizations"
+tags:
+  - APIs
+  - NerdGraph
+  - Examples
+metaDescription: "New Relic NerdGraph API features available for multi-tenant organizations." 
+redirects:
+---
+
+For multi-tenant New Relic organizations, we provide various features that let you manage member organizations. This tutorial will provide you with details on how to use [our NerdGraph API](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph) to manage multi-tenancy features. For general information on multi-tenancy features, see [Multi-tenant organizations](/docs/accounts/accounts-billing/account-structure/multitenant-organizations).
+
+## Requirements [#requirements]
+
+Before using this API, read the requirements and explanations of features in [Multi-tenant organizations](/docs/accounts/accounts-billing/account-structure/multitenant-organizations).  
+
+Specific features can have specific permissions requirements. For details, see the specific calls below. 
+
+## Tips on terms and fields [#field-definitions]
+
+...
+
+Calls: 
+
+## Account sharing [#account-sharing]
+
+PLACEHOLDER TEXT...
+
+Account sharing allows an organization to share a managed account to another organization. Through account sharing, our customers benefit by maintaining tenancy boundaries to ensure data protection, while enabling collaboration between members from different organizations.
+
+For MSP partners, account sharing allows the MSP to monitor accounts on behalf of their end customer, and share those monitored accounts with them if they wish to view the data. This is particularly handy when joint troubleshooting. 
+
+For large enterprise customers, account sharing allows business units that have application dependencies to collaborate with one another.  
+
+**Managed account**: an account created by the organization. For example: If an account was created by Org A and shared with Org B, the account would be considered a managed account for Org A and a shared account for Org B. 
+
+Requirements for the Shared Accounts API: 
+* Requires multi-tenant entitlement
+* Requires 4 capabilities created and assigned to the Organization Manager role. (pending tenant manager role) 
```

Comment:
just a note for Rob: we don't doc these granular things: we just say 'Requires the organization administration settings', or whatever